package.json

```diff
@@ -24,7 +24,7 @@
     "start": "pm2-runtime start pm2.config.js",
     "heroku-postbuild": "echo 'Skipping build'",
     "build": "node scripts/prod.js",
-    "clean": "rm -rf ./packages/client/__generated__ && rm ./queryMap.json && rm ./packages/server/graphql/public/schema.graphql && rm ./packages/server/graphql/private/schema.graphql && rm -rf ./dev && rm -rf ./dist && rm -rf ./build",
+    "clean": "rm -rf ./packages/client/__generated__; rm ./queryMap.json; rm ./packages/server/graphql/public/schema.graphql; rm ./packages/server/graphql/private/schema.graphql; rm -rf ./dev; rm -rf ./dist; rm -rf ./build",
```

Comment:
+1 Any reason not to combine everything into one `rm` command?

Reply:
Nope! That makes way more sense, changed to just one `rm -rf` command